ClassCastException: org.quartz.simpl.RAMJobStore cannot be cast to org.quartz.spi.JobStore

**Description:**
I am using the latest build siddhi-io-file-2.0.17-SNAPSHOT.
Observed the following error log when trying to read a CSV file, using Siddhi File Source.

Error log
```
[2021-08-28 19:31:13,883] ERROR {org.wso2.carbon.connector.framework.server.polling.PollingTaskRunner} - Exception occurred while scheduling job org.quartz.SchedulerException: JobStore class 'org.quartz.simpl.RAMJobStore' could not be instantiated. [See nested exception: java.lang.ClassCastException: org.quartz.simpl.RAMJobStore cannot be cast to org.quartz.spi.JobStore]
	at org.quartz.impl.StdSchedulerFactory.instantiate(StdSchedulerFactory.java:869)
	at org.quartz.impl.StdSchedulerFactory.getScheduler(StdSchedulerFactory.java:1559)
	at org.wso2.carbon.connector.framework.server.polling.PollingTaskRunner.start(PollingTaskRunner.java:73)
	at org.wso2.carbon.connector.framework.server.polling.PollingServerConnector.start(PollingServerConnector.java:57)
	at org.wso2.transport.file.connector.server.FileServerConnector.start(FileServerConnector.java:64)
	at io.siddhi.extension.io.file.FileSource.lambda$deployServers$7(FileSource.java:911)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassCastException: org.quartz.simpl.RAMJobStore cannot be cast to org.quartz.spi.JobStore
	at org.quartz.impl.StdSchedulerFactory.instantiate(StdSchedulerFactory.java:867)
	... 8 more
```

**Affected Product Version:**
siddhi-io-file-2.0.17-SNAPSHOT.

**OS, DB, other environment details and versions:**   
SI-tooling-1.1.0 with siddhi-io-file-2.0.17-SNAPSHOT. 

**Steps to reproduce:**
1. Create a CSV file with below content on  local file system.
``` 
{"event":{"symbol":"foo"}
```

2. Create the below Siddhi app on SI-tooling . Set the location of the CSV file you created in previous step in the `file.uri` parameter. Then start the app.
```
@App:name('ManageProductionStats')

@source(type = 'file', mode = "LINE", file.uri = "file:/Users/user/data/product.csv", tailing = "true",
    @map(type = 'csv'))
define stream SweetProductionStream (name string, amount double);

@sink(type='log')
define stream TotalProductionStream (name string, amount double);

from SweetProductionStream 
select name, sum(amount) as amount 
group by name
insert into TotalProductionStream;
```
Expected behaviour:
The following log should get printed.
``` 
[2021-09-06 22:14:31,603]  INFO {io.siddhi.core.stream.output.sink.LogSink} - ManageProductionStats : TotalProductionStream : Event{timestamp=1630946671603, data=[foo], isExpired=false}
``` 
Actual behaviour:
Following error gets printed.
``` 
ClassCastException: org.quartz.simpl.RAMJobStore cannot be cast to org.quartz.spi.JobStore
``` 
**Related Issues:**
https://github.com/siddhi-io/siddhi-io-file/issues/142

--- a/component/pom.xml
+++ b/component/pom.xml
@@ -243,7 +243,6 @@
                             META-INF=target/classes/META-INF,
                             META-INF/vfs-providers.xml=../resources/vfs-providers.xml
                         </Include-Resource>
-                        <DynamicImport-Package>*</DynamicImport-Package>
                     </instructions>
                 </configuration>
             </plugin>

